tools/python/util/convert_onnx_models_to_ort.py

```diff
@@ -12,35 +13,44 @@
 from .onnx_model_utils import get_optimization_level
 
 
+class OptimizationStyle(enum.Enum):
+    Fixed = 0
+    Runtime = 1
+
+
 def _path_match_suffix_ignore_case(path: typing.Union[pathlib.Path, str], suffix: str):
     if not isinstance(path, str):
         path = str(path)
     return path.casefold().endswith(suffix.casefold())
 
 
-def _onnx_model_path_to_ort_model_path(onnx_model_path: pathlib.Path, optimization_level_str: str):
+def _onnx_model_path_to_ort_model_path(onnx_model_path: pathlib.Path,
+                                       optimization_level_str: str,
+                                       optimization_style: typing.Optional[OptimizationStyle] = None):
     assert onnx_model_path.is_file() and _path_match_suffix_ignore_case(onnx_model_path, ".onnx")
-    return onnx_model_path.with_suffix(".{}.ort".format(optimization_level_str))
+    suffix = ".{}{}.ort".format(optimization_level_str,
+                                ".with_runtime_opt" if optimization_style == OptimizationStyle.Runtime else "")
+    return onnx_model_path.with_suffix(suffix)
```

Comment:
Can you please update \tools\python\util\check_onnx_model_mobile_usability.py as the documentation there talks about 'basic' and 'all' config files being generated?

Reply:
updated log messages there